README.md

```diff
@@ -239,11 +239,13 @@ Instead of using the field names, the export will use the labels as they are def
 
 Filters can automatically be added to the header row by setting `xlsx_auto_filter = True`. The filter will include all header columns in the worksheet.
 
-### Ignore fields
+### Specify or ignore fields
 
-By default, all fields are exported, but you might want to exclude some fields from your export. To do so, you can set an array with fields you want to exclude: `xlsx_ignore_headers = [<excluded fields>]`.
+By default, all fields are exported. However, this behavior can be changed.
 
-This also works with nested fields, separated with a dot (i.e. `icon.url`).
+To include only a specified list of fields, provide them with: `xlsx_specify_headers = [<fields to include>]`. Conversely, to exclude certain fields from your export, provide them with: `xlsx_ignore_headers = [<excluded fields>]`.
```

Comment:
I'm wondering whether these 2 options should be mutually exclusive: you could use one option or the other, but not both...

I looked at [the Django model forms API](https://docs.djangoproject.com/en/5.2/ref/forms/models/) with `fields` and `exclude` which have a similar concern and they're not mutually exclusive:

> `exclude` is an optional list of field names. If provided, the named fields will be excluded from the returned fields, even if they are listed in the `fields` argument.

Which -I think- matches the current implementation here. Maybe add a sentence to explain that ignore takes precendence over specify?